src/test/kotlin/carracing/domain/RaceTest.kt

```diff
@@ -0,0 +1,33 @@
+package carracing.domain
+
+import carracing.util.alwaysMoveGenerator
+import carracing.util.neverMoveGenerator
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class RaceTest : StringSpec({
+
+    lateinit var cars: List<Car>
+
+    beforeTest {
+        cars = List(3) { Car() }
+    }
+
+    "start should move cars" {
+        val race = Race(cars, 5, alwaysMoveGenerator())
```

Comment:
이번 기회에 [functional interface](https://kotlinlang.org/docs/fun-interfaces.html)를 학습하고 적용해보면 어떨까요? 🙂 

```suggestion
        val race = Race(cars, 5, RandomGenerator { 숫자 })
```

```kotlin
fun interface RandomGenerator {
    fun generate(): Int
}
```